dnsapi/dns_acmeproxy.sh

```diff
@@ -46,15 +46,6 @@ _acmeproxy_request() {
     return 1
   fi
 
-  ## Check for the credentials
```

Comment:
Can you add a message here ?  something like: `_info "ACMEPROXY_USERNAME" is not set , try to use ip based authentication.`